Monika After Story/game/script-fun-facts.rpy

```diff
@@ -757,3 +757,30 @@ label mas_fun_fact_cloud_weight:
     #Call the end
     call mas_fun_facts_end
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_fun_facts_database,
+            eventlabel="mas_fun_fact_coffeelegends",
+        ),
+        code="FFF"
+    )
+
+label mas_fun_fact_coffeelegends:
+    m 3eub "The last time I had a cup of coffee, I got a little curious about where it was first discovered."
+    m 3euc "You see, the use of coffee is recorded consistently around the 15th century, but...{w=0.2}it's unclear {i}how{/i} exactly it was found. There are quite a few legends claiming to be the very first."
```

Comment:
```suggestion
    m 3euc "You see, the use of coffee has been recorded consistently since around the 15th century, but...{w=0.2}it's unclear {i}how{/i} exactly it was found. There are quite a few legends claiming to be the very first."
```
I'm not sure if using the simple tense here is straight up wrong, but I feel like perfect continuous fits better.